pkg/server/backuprestoreserver.go

```diff
@@ -190,11 +192,21 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 	if runServerWithSnapshotter {
 		snapstoreConfig = b.config.SnapstoreConfig
 	}
-	etcdInitializer := initializer.NewInitializer(restoreOpts, snapstoreConfig, b.logger.Logger)
+	etcdInitializer := initializer.NewInitializer(restoreOpts, snapstoreConfig, b.config.EtcdConnectionConfig, b.logger.Logger)
 
 	handler := b.startHTTPServer(etcdInitializer, b.config.SnapstoreConfig.Provider, b.config.EtcdConnectionConfig, nil)
 	defer handler.Stop()
 
+	// Promotes member if it is a learner
+	backoff := retry.DefaultBackoff
+	backoff.Steps = 20 //TODO: find an apt value here
```

Comment:
Are we only trying for 20 times to call a `PromoteMember` ?

Reply:
Updated this to have continuously retry